### ISSUE TITLE
Error AsyncWithLDPRovider is not a function

### ISSUE THREAD

**Issue by SHND** [launchdarkly/jest-launchdarkly-mock#16]
I'm trying to mock the LaunchDarkly React SDK using this package. I'm using `AsyncWithLDPRovider`, but when I'm running my jest with react-library-testing I'm getting:

```
Error (0, _launchdarklyReactClientSdk.asyncWithLDProvider) is not a function
```
I believe this is fixed in https://github.com/launchdarkly-labs/jest-launchdarkly-mock/pull/15, but still don't see this in the latest npm package.

**Comment by yusinto** [comment]
We have an upcoming release that includes this fix. I'll update this issue when that's done. Thank you.

**Comment by SHND** [comment]
From the PR above, it seems that it is not the correct fix. I put a comment on the PR. I think `AsyncWithLDProvide` mock should return promise.

**Comment by yusinto** [comment]
Thank you for your patience. This is now fixed in [v1.0.0](https://github.com/launchdarkly/jest-launchdarkly-mock/releases/tag/1.0.0).
